Assets/QA/Tests/LegacyInputManager/Scripts/ShowIMUData.cs

```diff
@@ -0,0 +1,68 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+[RequireComponent(typeof(Text))]
+public class ShowIMUData : MonoBehaviour
```

Comment:
What does IMU stand for?

Reply:
IMU = Inertial motion unit.  It's the term for the physical modules that measure and report acceleration, gyroscope, and magnetometer (compass) data.  Does Unity have a better term for grouping all this data together, or is this your way of telling me that my class name can't have non-standard acronyms?